storage/cloud.py

```diff
@@ -1008,6 +1010,7 @@ def __init__(
     def get_direct_download_url(
         self, path, request_ip=None, expires_in=60, requires_cors=False, head=False, **kwargs
     ):
+        domain = self.cloudfront_distribution_domain
```

Comment:
minor nit. Can you move this line to 1038 that way all the `domain` logic is in one place